templates/user_detail.html

```diff
@@ -0,0 +1,93 @@
+{% extends "base-tw.html" %}
+
+{% block metatitle %}{{ user.name }} | OpenSAFELY Jobs{% endblock metatitle %}
+
+{% block breadcrumbs %}
+  {% url "home" as home_url %}
+
+  {% #breadcrumbs %}
+    {% breadcrumb title="Home" url=home_url %}
+    {% breadcrumb title=user.name active=True %}
+  {% /breadcrumbs %}
+{% endblock breadcrumbs %}
+
+{% block content %}
+  <div class="
+    mb-6 mt-3 flex flex-col gap-6 items-center text-center
+    md:mt-0 md:flex-row md:text-left
+  ">
+    <div class="flex-grow flex flex-col gap-y-4 md:gap-y-2">
+      <h1 class="text-3xl tracking-tight break-words md:text-4xl font-bold text-slate-900">
+        {{ user.name }}
+      </h1>
+    </div>
+  </div>
```

Comment:
This section can be replaced by the `article_header` component.

```suggestion
  {% article_header title=user.name class="mb-6" %}
```